green_cli/authenticators/__init__.py

```diff
@@ -178,19 +179,16 @@ def resolve(self, details):
             return response
         if details['action'] == 'sign_tx':
             return self.sign_tx(details)
-        if details['action'] == 'get_receive_address':
-            blinding_script_hash = bytes.fromhex(details['address']['blinding_script_hash'])
-            public_blinding_key = self.get_public_blinding_key(blinding_script_hash)
-            return json.dumps({'blinding_key': public_blinding_key.hex()})
+        if details['action'] == 'get_master_blinding_key':
+            return json.dumps({'master_blinding_key': self.self.master_blinding_key.hex()})
```

Comment:
is this reference to `self.self` right?

Reply:
No, last minute faffing. Will mark this PR as WIP until development/testing is complete.